components/CallToActionContent.vue

```diff
@@ -1,20 +1,38 @@
 <template>
-  <section class="container mx-auto text-center py-6 mb-12">
-    <h1 class="w-full my-2 text-5xl font-bold leading-tight text-center text-white">Call to Action</h1>
-    <div class="w-full mb-4">
-      <div class="h-1 mx-auto bg-white w-1/6 opacity-25 my-0 py-0 rounded-t"></div>
-    </div>
+  <section class="container call-to-action-content">
+    <the-title :isLight="true" label="Call to Action" />
 
-    <h3 class="my-4 text-3xl leading-tight">Main Hero Message to sell yourself!</h3>
+    <h3>Main Hero Message to sell yourself!</h3>
 
-    <button
-      class="mx-auto lg:mx-0 hover:underline bg-white text-gray-800 font-bold rounded-full my-6 py-4 px-8 shadow-lg"
-    >Action!</button>
+    <button>Action!</button>
   </section>
 </template>
 
 <script>
+import TheTitle from '@/components/TheTitle'
+
 export default {
-  name: 'CallToActionContent'
+  name: 'CallToActionContent',
+  components: {
+    'the-title': TheTitle
+  }
 }
 </script>
+
+<style scoped>
+.call-to-action-content {
+  @apply mx-auto text-center py-6 mb-12;
+}
+
+h3 {
+  @apply my-4 text-3xl leading-tight;
+}
+
+button {
+  @apply mx-auto bg-white text-gray-800 font-bold rounded-full my-6 py-4 px-8 shadow-lg;
+}
+
+button:hover {
```

Comment:
I believe we could also use `<button class="hover:underline">` in the template, what do you think?

Reply:
Sure, I think in this case we only have to consider where to slice between template and styling. Like, do all pseudo-classes go into the template? What about pseudo-classes that also use media queries?

In my opinion, it's just a better developer experience and readability to have everything in one place. Which is totally a personal thing I guess. :)

Reply:
I actually try to not have the <style> block anymore at all when using tailwindcss.

all pseudo-classes in template, all responsive-designs in the template.

I don't create media-queries in components anymore, I use the sm, md and lg of tailwind - or if I really need some changes, I add all of them inside the tailwind config, but never in the components